src/PresentationalComponents/Filters/CheckboxFilter.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { Checkbox } from '@patternfly/react-core';
+import PropTypes from 'prop-types';
+
+class CheckboxFilter extends Component {
+    state = {
+        checked: false
+    };
+
+    componentDidMount () {
+        this.props.filters[this.props.param] && this.props.filters[this.props.param].includes(this.props.value) ?
+            this.setState({ checked: true }) :
+            this.setState({ checked: false });
+    }
+
+    handleChange = checked => {
+        this.setState({ checked: !this.state.checked });
+        this.props.addRemoveFilters(
+            event.target.value,
```

Comment:
Event is missing in attributes. But it's not really needed you can access param and value from props.